src/main/java/org/breedinginsight/brapps/importer/model/imports/experimentObservation/ExperimentObservation.java

```diff
@@ -0,0 +1,272 @@
+    @ImportFieldMetadata(id="row", name="Row", description = "Horizontal (y-axis) position in 2D Cartesian space.")
+    private String row;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="column", name="Column", description = "Vertical (x-axis) position in 2D Cartesian space.")
+    private String column;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="treatmentFactors", name="Exp Treatment Factor Name", description = "treatement factors in an experiment with applied variables, like fertilizer or water regimens.")
+    private String treatmentFactors;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="ObsUnitID", name="Observation Unit ID", description = "A database generated unique identifier for experimental observation units")
+    private String ObsUnitID;
+
+    public BrAPITrial constructBrAPITrial(Program program, boolean commit, String referenceSource, UUID id) {
+        BrAPIProgram brapiProgram = program.getBrapiProgram();
+        BrAPITrial trial = new BrAPITrial();
+        if( commit ){
+            trial.setTrialName( Utilities.appendProgramKey(getExpTitle(), program.getKey() ));
+
+            // Set external reference
+            trial.setExternalReferences(getBrAPIExternalReferences(program, referenceSource, id, null, null));
+        }
+        else{
+            trial.setTrialName( getExpTitle() );
+        }
+        trial.setTrialDescription(getExpDescription());
+        trial.setActive(true);
+        trial.setProgramDbId(brapiProgram.getProgramDbId());
+        trial.setProgramName(brapiProgram.getProgramName());
+
+        trial.putAdditionalInfoItem("defaultObservationLevel", getExpUnit());
+        trial.putAdditionalInfoItem("experimentType", getExpType());
+
+        return trial;
+    }
+
+    public BrAPILocation constructBrAPILocation() {
+        BrAPILocation location = new BrAPILocation();
+        location.setLocationName(getEnvLocation());
+        return location;
+    }
+
+    public BrAPIStudy constructBrAPIStudy(
+            Program program,
+            boolean commit,
+            String referenceSource,
+            String expSequenceValue,
+            UUID trialId,
+            UUID id) {
+        BrAPIStudy study = new BrAPIStudy();
+        if ( commit ){
+            study.setStudyName(Utilities.appendProgramKey(getEnv(), program.getKey(), expSequenceValue));
+
+            // Set external reference
+            study.setExternalReferences(getBrAPIExternalReferences(program, referenceSource, trialId, id,null));
+        }
+        else {
+            study.setStudyName(getEnv());
+        }
+        study.setActive(true);
+        study.setStudyType(getExpType());
+        study.setLocationName(getEnvLocation());
+        study.setTrialName(getExpTitle());
+        study.setSeasons( List.of( getEnvYear()==null ? "" : getEnvYear() ) );
```

Comment:
I plan to keep this as it is...
setSeasons() requires a list of Strings.  List.of() will not except a null.  The current code fulfills those specifications.

Reply:
In that case though, aren't we going to have seasons as:

```
"seasons": [
      ""
    ]
```

In some cases? It would be better if there were no seasons to have an empty list so that it's clear there are no seasons. The empty string could be mistaken for something else.

```
"seasons": []
```